.github/workflows/performance.yml

```diff
@@ -119,6 +121,9 @@ jobs:
       - name: Install npm dependencies
         run: npm ci
 
```

Comment:
Not necessarily a blocker, but I wonder if we could cache these binaries with actions/cache to reduce the time to fetch these with identical configurations?

Reply:
Caching browser binaries is discouraged by Playwright.

Refs:

* https://playwright.dev/docs/ci#caching-browsers
* https://github.com/microsoft/playwright/pull/23814

Reply:
Works for me!